File: Assignment/Module-1(fundamentals/p7.cpp
#include <stdio.h>
main() 
{
    float celsius, fahrenheit;
    printf("Enter temperature in degrees Celsius: ");
    scanf("%f", &celsius);
    fahrenheit = (celsius * 9 / 5) + 32;
    printf("%.2f degrees Celsius is equal to %.2f degrees Fahrenheit.\n", celsius, fahrenheit);

}

File: Assignment/flutter dart assignment module-1/p7.dart
import 'dart:io';

void main() {
  double celsius, fahrenheit;

  stdout.write("Enter temperature in degrees Celsius: ");
  celsius = double.parse(stdin.readLineSync()!);

  fahrenheit = (celsius * 9/5) + 32;

  print("Temperature in Fahrenheit: $fahrenheit°F");
}
